backend/director/llm/minimax.py

```
@@ -0,0 +1,200 @@
+            if tool.get("name")
+        ]
+
+    @staticmethod
+    def _strip_think_tags(content: str) -> str:
+        """Strip <think>...</think> tags from MiniMax M2.7 reasoning output."""
+        if not content:
+            return content
+        return re.sub(r"<think>.*?</think>\s*", "", content, flags=re.DOTALL).strip()
+
+    def chat_completions(
+        self, messages: list, tools: list = [], stop=None, response_format=None
```

Ruff (0.15.6)

[warning] Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)